scripts/Muon/FFT_view.py

```diff
@@ -0,0 +1,119 @@
+from __future__ import (absolute_import, division, print_function)
+
+from PyQt4 import QtCore, QtGui
+from Muon import widget_helper
+
+
+class FFTView(QtGui.QWidget):
+    # signals
+    buttonSignal = QtCore.pyqtSignal()
+    tableClickSignal = QtCore.pyqtSignal(object,object)
+
+    def __init__(self, parent=None):
+        super(FFTView, self).__init__(parent)
+        self.grid = QtGui.QGridLayout(self)
+        self.table 	= QtGui.QTableWidget(self)
```

Comment:
Please watch spacing around assignments, it's a bit inconsistent over these files. It would be better to stick to one style of a single space either side of the `=` operator